### ISSUE TITLE
Missing batchnorm and relu after this layer, same problem in other models defs

### ISSUE THREAD

**Issue by vgangal101** [vgangal101/pytorch_resnet_implementation#3]
https://github.com/vgangal101/pytorch_resnet_implementation/blob/f0b34ed78ad8d6a2d5f27aa56f0b8b2c9cd18e09/model_arch.py#L93

Missing batchnorm and relu after this layer
